sharepoint/sharepoint-ps/sharepoint-online/Export-SPODataAccessGovernanceInsight.md

```diff
@@ -0,0 +1,64 @@
+---
+external help file: Microsoft.Online.SharePoint.PowerShell.dll-Help.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version:
+schema: 2.0.0
+---
+
+# Export-SPODataAccessGovernanceInsight
+
+## SYNOPSIS
+
+This commands exports/downloads the DAG report to the default path "C:\WINDOWS\system32\"
+
+## SYNTAX
+
+```
+Export-SPODataAccessGovernanceInsight -ReportID <Guid> [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+This commands exports/downloads the DAG report, specified by the ReportID, to the default path "C:\WINDOWS\system32\". The ReportID is shown in the output of the 'Start-SPODataAccessGovernanceInsight' command. It can also be fetched from the output of the 'Get-SPODataAccessGovernanceInsight' command.
+
+## EXAMPLES
+
+### Example 1
+
+```powershell
+PS C:\> Export-SPODataAccessGovernanceInsight -ReportID 28f4c550-215a-472b-a123-c11e5fa8804c
+```
+
+This command downloads the report of the given ID to the default path "C:\WINDOWS\system32\"
```

Comment:
"In this example, the DAG report with the given ID is exported and downloaded to the current working directory."